packages/eslint-config-small-ads/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "eslint-config-small-ads",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {},
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.14.0",
+    "@typescript-eslint/parser": "^4.14.0",
+    "eslint": "^7.18.0",
+    "eslint-config-airbnb-typescript": "^12.0.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-config-react": "^1.1.7",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-react": "^7.22.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "prettier": "^2.2.1",
+    "typescript": "^4.1.3"
```

Comment:
Also if we move tsconfig to each packahe I think typescript isnt needed as a dependency here.

Reply:
If I delete Typescript I get this error whenever I try to run the projects.
```
Failed to load parser '@typescript-eslint/parser' declared in '.eslintrc.js » eslint-config-small-ads': Cannot find module 'typescript'
Require stack:
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/packages/eslint-config-small-ads/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/packages/eslint-config-small-ads/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/packages/eslint-config-small-ads/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/packages/eslint-config-small-ads/node_modules/@typescript-eslint/parser/dist/index.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/@eslint/eslintrc/lib/config-array-factory.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/@eslint/eslintrc/lib/index.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/eslint/lib/cli-engine/cli-engine.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/eslint/lib/cli-engine/index.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/eslint/lib/api.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/eslint-webpack-plugin/dist/getESLint.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/eslint-webpack-plugin/dist/linter.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/eslint-webpack-plugin/dist/index.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/eslint-webpack-plugin/dist/cjs.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/react-scripts/config/webpack.config.js
- /home/dci/learningToProgram/Web-Development/projects/with-christos/core/src/fe-small-advertisements/node_modules/react-scripts/scripts/start.js
```

Reply:
Will doublecheck.

Reply:
You right about this